shared/message/platform/gazebo/Torso.proto

```diff
@@ -24,16 +24,15 @@ import "Matrix.proto";
 import "Vector.proto";
 
 message Torso {
-    // The name of the model in the simulation that this position is for
     string model = 1;
```

Comment:
This is the only one without `/// The name of the model in the simulation that these servo targets are for`

Reply:
FIX IT